src/NUnitFramework/tests/Attributes/SingleThreadedFixtureTests.cs

```diff
@@ -40,10 +40,10 @@ public void TestRunsOnSameThreadAsParent()
             Assert.That(Thread.CurrentThread, Is.EqualTo(ParentThread));
         }
 
-        [Test]
-        public void TestWithTimeoutIsInvalid()
+        [Test, Timeout(100)]
+        public void TestWithTimeoutIsValid()
         {
-            CheckTestIsInvalid<SingleThreadedFixture_TestWithTimeout>("TimeoutAttribute may not be specified");
+            Assert.That(Thread.CurrentThread, Is.EqualTo(ParentThread));
```

Comment:
This is now the same as `TestWithTimeoutRunsOnSameThread()` below - do we need both?